### ISSUE TITLE
Display OS SW version in device node

### ISSUE THREAD

**Issue by cbrake** [simpleiot/simpleiot#324]
cc @kraj

**Comment by cbrake** [comment]
this should just be a point so that we can track software version changes over time automatically.

**Comment by cbrake** [comment]
Yoe images have OS version in VERSION_ID field:

```
root@raspberrypi4-64:~# more /etc/os-release 
ID=yoe
NAME="Yoe Linux"
VERSION="2022.03 (Orkney)"
VERSION_ID=0.0.1
PRETTY_NAME="Yoe Linux 2022.03 (Orkney)"
DISTRO_CODENAME="Orkney"
HOME_URL="https://github.com/YoeDistro"
SUPPORT_URL="https://github.com/YoeDistro/yoe-distro/issues"
BUG_REPORT_URL="https://github.com/YoeDistro/yoe-distro/issues"
```

**Comment by cbrake** [comment]
Perhaps we should make which variable we use in `/etc/os-release` configurable though through an env variable. That way the user can use whatever fits the system they are running on.

**Comment by kraj** [comment]
> Perhaps we should make which variable we use in `/etc/os-release` configurable though through an env variable. That way the user can use whatever fits the system they are running on.

there is [os-release](https://github.com/YoeDistro/openembedded-core/blob/master/meta/recipes-core/os-release/os-release.bb) recipe which we add a [bbappend](https://github.com/YoeDistro/yoe-distro/blob/master/sources/meta-yoe/recipes-core/os-release/os-release.bbappend) in meta-yoe, we could certainly document setting these variables via config metadata if someone wants to do so.

**Comment by cbrake** [comment]
![image](https://user-images.githubusercontent.com/402813/163829093-14c0d644-243d-49e0-9c83-acc3c642c9ab.png)

**Comment by cbrake** [comment]
implemented in #353
